pygmt/figure.py

```diff
@@ -284,6 +284,38 @@ def show(self, dpi=300, width=500, method="static"):
             img = Image(data=png, width=width)
         return img
 
+    def shift_origin(self, xshift=None, yshift=None):
+        """
+        Shift plot origin in x and/or y directions.
+
+        This method shifts plot origin relative to the current origin by (*xshift*,*yshift*)
+        and optionally append the length unit (**c**, **i**, or **p**).
+
+        Prepend **a** to shift the origin back to the original position
+        after plotting, prepend **c** to center the plot on the center of the
+        paper (optionally add shift), prepend **f** to shift the origin relative
+        to the fixed lower left corner of the page, or prepend **r** [Default] to
+        move the origin relative to its current location.
+
+        Detailed usage at http://gmt.soest.hawaii.edu/doc/latest/GMT_Docs.html#plot-positioning-and-layout-the-x-y-options
```

Comment:
What's the best way to reference the official GMT documentation?

Reply:
I still don't know of a good way to do this yet. I'm open to suggestions. It would be best to have this as a variable somewhere and input into the docstring somehow.

Reply:
@leouieda Can we rename the `gmt_module_docs` decorator to something like `gmt_docs`, which supports following conversions:
```
{gmt_docs basemap} => http://gmt.soest.hawaii.edu/doc/latest/basemap.html
{gmt_docs gmt xy-full} => http://gmt.soest.hawaii.edu/doc/latest/gmt.html#xy-full
```

Reply:
I would actually like to get rid of most of those decorators in favor of something simpler to maintain. But I don't think there is a good way to format the docstring without using a decorator. We could certainly add that functionality. But we should be using the https://www.generic-mapping-tools.org/gmt/latest/ domain now.

Reply:
What about putting the raw links in the docstrings, without using a decorator or any substitutions? If someday the GMT documentation site changes, it's still easy to update all the links.